curvefs/src/mds/fs_info_wrapper.cpp

```diff
@@ -50,8 +49,13 @@ FsInfoWrapper::FsInfoWrapper(const ::curvefs::mds::CreateFsRequest* request,
     fsInfo.set_enablesumindir(request->enablesumindir());
     fsInfo.set_txsequence(0);
     fsInfo.set_txowner("");
+    // TODO(@lixiaocui1): Currently, curveadm does not support setting recycling
```

Comment:
I will add it, and is there a default value for `recyleTimeHour` if user not specified?

Reply:
recycleTimeHour is `optional` in `message FsInfo`. it doesn't have a default value.

Reply:
> recycleTimeHour is `optional` in `message FsInfo`. it doesn't have a default value.

Okay~ i will give a default value (1 hours) in curveadm.